app/controllers/book_suggestions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class BookSuggestionsController < ApplicationController
+  def create
+    book_suggestion = BookSuggestion.create(book_suggestion_params)
+    if book_suggestion.save
```

Comment:
You already save when use a create method, you can use `.valid?` and use `book_suggestion.errors.messages` to return the errors